### PR TITLE
Ab tabbed interface

### PR DESCRIPTION
**Pull request by aborov**

### CODE REVIEW

File: app/views/users/following.html.erb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      <li class="nav-item">
+        <%= link_to "Following", following_path(@user.username), class: "nav-link active" %>
+      </li>
+    </ul>
+
+    <div class="list-group mt-3">
+      <% @user.leaders.each do |leader| %>
+        <%= link_to leader.username, user_path(leader.username), { :class => "list-group-item list-group-item-action" } %>
+      <% end %>
+    </div>
```
Comment (jniewiel) [inline app/views/users/following.html.erb:29]: To save some space and make it easier, you can make a separate render page (_follow.html.erb) to handle both followers and leaders in your followers and following pages for User.

**Review by jniewiel — APPROVED** [review]
Nice job! I'd recommend just implementing that change I commented on your /users/show file.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#">Feed</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#">Followers</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#">Following</a>
+      </li>
```
[the remark this answers is not in the document]
Comment (aborov) [reply app/views/users/show.html.erb:22]: Thanks for the catch, I believe I just forgot to replace it here.

File: app/views/users/following.html.erb (hunk 1 above)
Comment (aborov) [reply app/views/users/following.html.erb:29]: Thanks, that's a smart suggestion. I will implement.

**Review by CarlosCastanedaDev — APPROVED** [review]
Great job, Aleksei!

In your "users" view folder you could also create a partial for all the tab info and use the partial in the feed, followers, following, liked and show view files. 

You can now merge these changes back to main.

**Review by aizatibraimova — APPROVED** [review]

**Pull request merged**